.pyproject_generation/pyproject_custom.toml

```
@@ -0,0 +1,16 @@
+[project]
+name = "pcs"
+version = "1.2.0"
+description = "Purge Controller Service - a service to commission file deletions"
+dependencies = [
+    "ghga-service-commons[api]>=2.0.0",
+    "ghga-event-schemas==1.0.0",
+    "hexkit[akafka]>=2.0.0",
```

```suggestion
    "ghga-service-commons[api]>=3.0.0",
    "ghga-event-schemas==3.0.0",
    "hexkit[akafka]>=2.1.0",
```

Should probably be part of this PR as it regenerates the requirement files.